octygon-button-styler.php

```diff
@@ -1,146 +1,171 @@
+				'oct_btn_shrink_4'    => __( 'Shrink 4' ),
 			),
-		) 
+		)
 	);
-	$wp_customize->add_control( 
-		'oct_btn_shadow_styles', 
+	$wp_customize->add_control(
+		'oct_btn_shadow_styles',
 		array(
-			'priority' 	=> 20, // Within the section.
-			'section' 	=> 'et_divi_buttons_hover', // Required, core or custom.
-			'settings' 	=> 'oct_btn_shadow_styles',
-			'label'    	=> __( 'Button Shadow Styles', 'oct' ), 
-			'type'     	=> 'radio',
-			'choices'  	=> array(
-				'oct_btn_shadow_none'   	=> __( 'None' ),
-                'oct_btn_shadow_1'  		=> __( 'Shadow 1' ),
-                'oct_btn_shadow_2'  		=> __( 'Shadow 2' ),
-                'oct_btn_shadow_3'  		=> __( 'Shadow 3' ),
-                'oct_btn_shadow_4'  		=> __( 'Shadow 4' )
+			'priority' => 20,
+			'section'  => 'et_divi_buttons_hover',
+			'settings' => 'oct_btn_shadow_styles',
+			'label'    => __( 'Button Shadow Styles', 'oct' ),
+			'type'     => 'radio',
+			'choices'  => array(
+				'oct_btn_shadow_none' => __( 'None' ),
+				'oct_btn_shadow_1'    => __( 'Shadow 1' ),
+				'oct_btn_shadow_2'    => __( 'Shadow 2' ),
+				'oct_btn_shadow_3'    => __( 'Shadow 3' ),
+				'oct_btn_shadow_4'    => __( 'Shadow 4' ),
 			),
-		) 
+		)
 	);
-	$wp_customize->add_control( 
-		'oct_btn_slide_styles', 
+	$wp_customize->add_control(
+		'oct_btn_slide_styles',
 		array(
-			'priority' 	=> 20, // Within the section.
-			'section' 	=> 'et_divi_buttons_hover', // Required, core or custom.
-			'settings' 	=> 'oct_btn_slide_styles',
-			'label'    	=> __( 'Button Slide Styles', 'oct' ), 
-			'type'     	=> 'radio',
-			'choices'  	=> array(
-				'oct_btn_slide_none'   		=> __( 'None' ),
-                'oct_btn_slide_left'  		=> __( 'Slide Left' ),
-                'oct_btn_slide_right'  		=> __( 'Slide Right' ),
-                'oct_btn_slide_top'  		=> __( 'Slide Top' ),
-                'oct_btn_slide_bottom'  	=> __( 'Slide Bottom' ),
-                'oct_btn_slide_out_h'  		=> __( 'Slide Out Horizontal' ),
-                'oct_btn_slide_out_v'  		=> __( 'Slide Vertical' )
+			'priority' => 20,
+			'section'  => 'et_divi_buttons_hover',
+			'settings' => 'oct_btn_slide_styles',
+			'label'    => __( 'Button Slide Styles', 'oct' ),
+			'type'     => 'radio',
+			'choices'  => array(
+				'oct_btn_slide_none'   => __( 'None' ),
+				'oct_btn_slide_left'   => __( 'Slide Left' ),
+				'oct_btn_slide_right'  => __( 'Slide Right' ),
+				'oct_btn_slide_top'    => __( 'Slide Top' ),
+				'oct_btn_slide_bottom' => __( 'Slide Bottom' ),
+				'oct_btn_slide_out_h'  => __( 'Slide Out Horizontal' ),
+				'oct_btn_slide_out_v'  => __( 'Slide Vertical' ),
 			),
-		) 
+		)
 	);
 }
 add_action( 'customize_register', 'oct_customize_register' );
 
-//Octygon Customizer Javascript & CSS Output
-function oct_customize_output()
-{
-	$oct_btn_grow = ( get_option( 'oct_btn_grow_styles' ) == 'oct_btn_grow_none' ) ? '' : get_option( 'oct_btn_grow_styles' );
```

Comment:
This logic is hard to grep, and a bit messy. Aim for clearness in your code, not brevity. I've reworked it to use multiple `if` clauses instead.